### DISCUSSION TITLE
Problem on MAC running Pshrink

### DISCUSSION THREAD

**Discussion by jestern77** [Drewsif/PiShrink#283] [General]
HI there I correctly installed PiShrink using the three command lines. When I run the code though it give me an error.

sudo pishrink.sh /Users/jestern/Desktop/Rpi.img 
/usr/local/bin/pishrink.sh: line 11: declare: -A: invalid option
declare: usage: declare [-afFirtx] [-p] [name[=value] ...]
/usr/local/bin/pishrink.sh: line 12: declare: -A: invalid option
declare: usage: declare [-afFirtx] [-p] [name[=value] ...]
/usr/local/bin/pishrink.sh: line 13: declare: -A: invalid option
declare: usage: declare [-afFirtx] [-p] [name[=value] ...]
pishrink.sh v0.1.4
pishrink.sh: ERROR occurred in line 257: parted is not installed.

Can you help me understand what is wrong? I did the image with diskutil on Mac from my Rapsberry pi.

**Comment by ghost** [comment]
Hey @jestern77 you can use [PiShrink dockerized](https://github.com/monsieurborges/pishrink-docker) | [Docker Hub PiShrink](https://hub.docker.com/r/monsieurborges/pishrink).

Hi @Drewsif, would it be possible to update the README and include a note about PiShrink for macOS? Thanks so much!

## macOS Setup

1. [Install Docker Desktop on Mac](https://docs.docker.com/docker-for-mac/install/)

2. Make a copy of a Raspberry Pi SD card that you want to shrink ([see instructions here](https://github.com/monsieurborges/raspberry-pi/blob/master/setup/clone-sd-card.md)).

3. Using the Terminal, access the directory containing the Raspberry Pi image:

```bash
cd ~/Directory-with-RPi-image
```

4. Run PiShrink dockerized:

```bash
docker run --privileged=true --rm \
    --volume $(pwd):/workdir \
    monsieurborges/pishrink \
    pishrink -Zv IMAGE_NAME.img NEW-IMAGE_NAME.img
```

**Reply by Drewsif** [reply]
Working on it 😄. I'm going to use your docker container as a base and put one in the repo as well. I'm planning on linking to your cloning directions as well

**Comment by Drewsif** [comment]
Instructions in so closing this! Check the README for the latest
